src/snowflake/cli/api/project/schemas/project_definition.py

```diff
@@ -209,11 +212,13 @@ def _merge_mixins_with_entity(
         for mixin_name in entity_mixins_names:
             if mixin_name not in mixin_defs:
                 raise ValueError(f"Mixin {mixin_name} not defined")
-
+        context = get_cli_context()
         # Build object override data from mixins
         data: dict = {}
         for mx_name in entity_mixins_names:
-            data = cls._merge_data(data, mixin_defs[mx_name])
+            if mx_name not in cls._applied_mixins.default.get(entity_id, []):  # type: ignore
```

Comment:
What is `cls._applied_mixins.default`? Isn't `cls._applied_mixins` a dictionary?

Reply:
Pydantic wraps it in its' own `ModelPrivateField` class, that needs using this access methods

Reply:
Added docstring in attribute declaration